geometry_msgs/Point position
uint8[] state
